[initState

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
Please remove this file.

Reply:
Hi @v-viyada removed it, please re-review